### PR TITLE
Refresh icons with the latest from GXD

### PR DESCRIPTION
**Pull request by drewcovi**
## Motivation

Our VP of GXD sent a small request to get the icon aligned for the time being. It's pretty straight forward in design, but definitely more tailored than the current logo.

## Approach

I leveraged the app-icon [https://github.com/dwmkerr/app-icon](url) `npm` package. Let me know if we should add this as a dependency. It certainly streamlines asset creation.

#### TODOS and Open Questions

Let me know if we want to bundle this tool in the repository, it's a small addition but it would help ensure we have a standardized approach to updating app icons in the future.

### Before
Screen Shot 2019-05-06 at 2 25 20 PM

### After
Screen Shot 2019-05-06 at 2 25 53 PM

### CODE REVIEW

**Comment by cherewaty** [comment]
@drewcovi I think we want all of the PNGs to have the background fill to the canvas' right angles instead of attempting to replicate the mask. That way any Android flavor will deal (https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive) and the two different iOS containing shapes (round rect pre-iOS 10, squircle after) will all fill perfectly.

File: App_Resources/Android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,7 +21,8 @@
 	<application
 		android:name="com.tns.NativeScriptApplication"
 		android:allowBackup="true"
-		android:icon="@drawable/icon"
+		android:icon="@mipmap/ic_launcher"
+    android:roundIcon="@mipmap/ic_launcher_round"
```
Comment (cowboyd) [inline App_Resources/Android/src/main/AndroidManifest.xml]: Looks like this is slightly mis-aligned, I don't think it's a nit worth picking, but we should add `xml` and `json` linting to our linting process.

**Comment by cherewaty** [comment]
I added the `release-to-beta` label so this goes out right away to everyone.

**Comment by cowboyd** [comment]
> I added the `release-to-beta` label so this goes out right away to everyone.

Slow off the gun, I am.

**Comment by drewcovi** [comment]
@cherewaty re-requesting a review. screenshots attached, and resolved adaptive icon issues.

File: App_Resources/Android/src/main/AndroidManifest.xml (hunk 1 above)
Comment (drewcovi) [reply App_Resources/Android/src/main/AndroidManifest.xml]: Should be obsolete with the latest updates.

**Pull request merged**